test/integration/exasol/test_get_metadata_functions.py

```diff
@@ -220,26 +188,23 @@ def test_get_view_definition_view_name_none(self, engine_name):
             pytest.param("NOT_A_SCHEMA", "s", id="not existing schema"),
         ],
     )
-    def test_get_columns_raises_exception_for_no_table(
+    def test_get_columns_raises_exception_for_non_existing_entities(
```

Comment:
Why is the test called "entities"? Are there other entities besides tables that could be non-existing and still be queried about `get_columns()`?